app/src/main/java/org/septa/android/app/MainActivity.java

```diff
@@ -365,6 +418,140 @@ public void goToSchedulesForTarget(StopModel start, StopModel destination, Trans
         switchToSchedules(bundle);
     }
 
+    @Override
```

Comment:
There is a lot of logic in this MainActvity around the database expansion.  Encapsulate this logic in its own class.

Reply:
done